### DISCUSSION TITLE
Mason installs vtsls instead of typescript-language-server . How do i force it to use tsserver instead?

### DISCUSSION THREAD

**Discussion by RHOOPH** [LazyVim/LazyVim#3524] [Q&A] [answered]
I recently updated lazyvim, I have been having trouble with formatting with prettier.

Formatting of the jsx is not working  as before, My guess is it is because of vtsls. 

Anyone else facing this issue ?

**Comment by dpetka2001** [comment, marked as answer]
Formatting with prettier has been changed to require a prettier configuration file at the root of your project. If it doesn't find one then it doesn't format. If you want to revert to the old behavior that it always formats even without the existence of a prettier config file, then you can put in your `/lua/config/options.lua` file the option `vim.g.lazyvim_prettier_needs_config = false`.
